test/db/analysis/vars

```diff
@@ -954,112 +885,409 @@ s dbg.feraiseexcept
 .(sig)
 s dbg._Atrealloc
 .(sig)
+s dbg.fn1
+.(sig)
+s dbg.__ltdf2
+.(sig)
 EOF
 EXPECT=<<EOF
-var va_list ap @ stack + 0x4
-arg const char *fmt @ a6
-int printf(const char *format);
+var int ans @ empty
+var int32_t arg5 @ a4
+arg const char *fmt @ loclist: [
+	(0x80000c22, 0x80000c2c) [DW_OP_reg20 20] a4
+	(0x80000c2c, 0x80000c3b) [DW_OP_reg22 22] a6
+	(0x80000c3b, 0x80000c3c) [DW_OP_GNU_entry_value 0x64, DW_OP_stack_value] <evaluation waiting>
+	]
+var va_list ap @ loclist: [
+	(0x80000c22, 0x80000c36) [DW_OP_fbreg 0x0, DW_OP_stack_value] CFA+0
+	(0x80000c36, 0x80000c3b) [DW_OP_reg23 23] a7
+	(0x80000c3b, 0x80000c3c) [DW_OP_fbreg 0x0, DW_OP_stack_value] CFA+0
+	]
+int printf(const char *fmt, ...);
             ; CALL XREFS from dbg.main @ 0x8000054e, 0x80000636
             ;-- printf:
-/ int dbg.printf(const char *fmt, va_args ..);
-|           ; arg const char *fmt @ a6
-|           ; var va_list ap @ stack + 0x4
-|           0x80000c22      mov.aa a6, a4                              ; printf.c:10 ; arg5 ; int printf(const char *fmt, va_args ..);
+/ int printf(const char *fmt, ...)
+|           ; var int32_t arg5 @ a4
+|           ; arg const char *fmt @ loclist: [
```

Comment:
Regarding these loclists, could you please use a short form I suggested in https://github.com/rizinorg/rizin/pull/3565#discussion_r1278833844
```
var va_lost ap @ [<regname1|stackoffset1>, <regname2|stackoffset2>, ...]
```

Reply:
It's just that now we have a lot of restrictions on variable name substitution, in which case our output will look a bit strange if we omit Range.begin

Reply:
i'm not super sure if these are useful on output.

Reply:
Another option we could simply omit everything except stack and registers for storage in this variable list, to preserve the old style output, but to show these vars anyway

Reply:
We can try to improve output and substitution in later PRs. What do you think?

Reply:
For more detailed output we will have to rework `afv` commands as well: https://github.com/rizinorg/rizin/issues/3702

Reply:
> If there is a way to select biggest range as you suggested - use it if it's register or stack, if you don't know what to show in the `@ ...` part, simply omit it @imbillow

OK